### PR TITLE
update docs versions for Insights, Console, Cost & Subscriptions

### PR DESCRIPTION
**Pull request by ryandeussing**
## PR Title :boom:

Where links to Insights, Console, Cost & Subscriptions docs exist, this PR updates the url to the 2023 version
Suggested formats: 

1. Fixes [RHIROS-982 ](https://issues.redhat.com/browse/RHIROS-982) 

Feel free to remove this section from PR description once done.

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here.

## Documentation requires update? :memo:

- [ ] Yes
- [ ] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

**Review by PreetiW — CHANGES_REQUESTED** [review]
Thank you for the PR @ryandeussing 🙌🏻 
I see the [jobs are failing](https://ci.int.devshift.net/job/RedHatInsights-ros-frontend-pr-check/125/console), can you please run the lint `npm run lint` on the PR and fix the issues using `lint:js:fix`? Let me know if you have any questions! 💯

File: src/constants.js [hunk 1]
```diff
@@ -67,94 +77,107 @@ export const WITH_WAITING_FOR_DATA_PARAM = 'with_waiting_for_data';
...
+  'Apr',
+  'May',
+  'Jun',
+  'Jul',
+  'Aug',
+  'Sep',
+  'Oct',
+  'Nov',
+  'Dec',
+];
```
Comment (kgaikwad) [inline src/constants.js:180]:
```suggestion
export const MONTHS = [
    'Jan',
    'Feb',
    'Mar',
    'Apr',
    'May',
    'Jun',
    'Jul',
    'Aug',
    'Sep',
    'Oct',
    'Nov',
    'Dec'
];
```

File: src/constants.js [hunk 2]
```diff
@@ -67,94 +77,107 @@ export const WITH_WAITING_FOR_DATA_PARAM = 'with_waiting_for_data';
...
+    key: 'report_date',
+    title: 'Last reported',
+    modalTitle: 'Last reported',
+    renderFunc: (data) => displayLastReported(data),
+    isChecked: true,
+    isDisabled: false,
+    isShownByDefault: true,
+    transforms: [nowrap],
+    cellTransforms: [nowrap],
+  },
```
Comment (kgaikwad) [inline src/constants.js:163]:
```suggestion
    {
        key: 'display_name',
        title: 'Name',
        modalTitle: 'Name',
        renderFunc: (data, id, item) => systemName(data, id, item),
        isChecked: true,
        isDisabled: true,
        isShownByDefault: true
    },
    {
        key: 'os',
        title: (
            <Tooltip content={<span>Operating system</span>}>
                <span>OS</span>
            </Tooltip>
        ),
        modalTitle: 'Operating system',
        dataLabel: 'Operating system',
        renderFunc: (data) => displayOS(data),
        isChecked: true,
        isDisabled: false,
        isShownByDefault: true
    },
    {
        key: 'performance_utilization.cpu',
        title: 'CPU utilization',
        modalTitle: 'CPU utilization',
        renderFunc: (data, id, item) => scoreProgress(data, id, item),
        isChecked: true,
        isDisabled: false,
        isShownByDefault: true
    },
    {
        key: 'performance_utilization.memory',
        title: 'Memory utilization',
        modalTitle: 'Memory utilization',
        renderFunc: (data, id, item) => scoreProgress(data, id, item),
        isChecked: true,
        isDisabled: false,
        isShownByDefault: true
    },
    {
        key: 'performance_utilization.max_io',
        title: (
            <Tooltip content={<span>IOPS</span>}>
                <span>I/O utilization</span>
            </Tooltip>
        ),
        modalTitle: 'I/O utilization',
        dataLabel: 'I/O utilization',
        renderFunc: (data, id, item) => diskUsageData(data, id, item),
        isChecked: true,
        isDisabled: false,
        isShownByDefault: true
    },
    {
        key: 'number_of_suggestions',
        title: 'Suggestions',
        modalTitle: 'Suggestions',
        renderFunc: (data, id, item) => recommendations(data, id, item),
        isChecked: true,
        isDisabled: false,
        isShownByDefault: true
    },
    {
        key: 'state',
        title: 'State',
        modalTitle: 'State',
        renderFunc: (data) => displayState(data),
        isChecked: true,
        isDisabled: false,
        isShownByDefault: true
    },
    {
        key: 'report_date',
        title: 'Last reported',
        modalTitle: 'Last reported',
        renderFunc: (data) => displayLastReported(data),
        isChecked: true,
        isDisabled: false,
        isShownByDefault: true,
        transforms: [nowrap],
        cellTransforms: [nowrap]
    }
```

File: src/constants.js [hunk 3]
```diff
@@ -22,42 +31,43 @@ export const NO_DATA_VALUE = 'N/A';
...
+          label: 'Undersized',
+          value: 'Undersized',
+        },
+        {
+          label: 'Waiting for data',
+          value: 'Waiting for data',
+        },
+      ],
+    },
+  },
```
Comment (kgaikwad) [inline src/constants.js:70]:
```suggestion
     state: {
        label: 'State',
        type: 'checkbox',
        filterValues: {
            items: [
                {
                    label: 'Idling',
                    value: 'Idling'
                },
                {
                    label: 'Optimized',
                    value: 'Optimized'
                },
                {
                    label: 'Oversized',
                    value: 'Oversized'
                },
                {
                    label: 'Under pressure',
                    value: 'Under pressure'
                },
                {
                    label: 'Undersized',
                    value: 'Undersized'
                },
                {
                    label: 'Waiting for data',
                    value: 'Waiting for data'
                }
            ]
        }
    }
```

File: src/constants.js [hunk 4]
```diff
@@ -22,42 +31,43 @@ export const NO_DATA_VALUE = 'N/A';
 
 // Getting started URL
 // eslint-disable-next-line max-len
-export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
+export const GETTING_STARTED_URL =
+  'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
```
Comment (kgaikwad) [inline src/constants.js:35]:
```suggestion
export const GETTING_STARTED_URL =  'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
```

File: src/constants.js [hunk 5]
```diff
@@ -1,8 +1,17 @@
 import React from 'react';
 import { nowrap } from '@patternfly/react-table';
 import { Tooltip } from '@patternfly/react-core';
-import { displayState, recommendations, scoreProgress, systemName } from './store/entitiesReducer';
-import { diskUsageData, displayLastReported, displayOS } from './Components/RosTable/RenderColumn';
+import {
+  displayState,
+  recommendations,
+  scoreProgress,
+  systemName,
```
Comment (kgaikwad) [inline src/constants.js:8]:
```suggestion
    displayState,
    recommendations,
    scoreProgress,
    systemName
```

File: src/constants.js [hunk 6]
```diff
@@ -1,8 +1,17 @@
...
+} from './store/entitiesReducer';
+import {
+  diskUsageData,
+  displayLastReported,
+  displayOS,
```
Comment (kgaikwad) [inline src/constants.js:13]:
```suggestion
    diskUsageData,
    displayLastReported,
    displayOS
```

**Review by kgaikwad — CHANGES_REQUESTED** [review]

File: src/constants.js [hunk 7]
```diff
@@ -22,11 +31,11 @@ export const NO_DATA_VALUE = 'N/A';
...
-export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
+export const GETTING_STARTED_URL =  'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
 
 // Custom Filters
 export const CUSTOM_FILTERS = {
-    state: {
+     state: {
         label: 'State',
         type: 'checkbox',
         filterValues: {
```
Comment (kgaikwad) [inline src/constants.js:41]:
```suggestion
    state: {
        label: 'State',
        type: 'checkbox',
        filterValues: {
```

File: src/constants.js [hunk 8]
```diff
@@ -22,7 +31,7 @@ export const NO_DATA_VALUE = 'N/A';
 
 // Getting started URL
 // eslint-disable-next-line max-len
-export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
+export const GETTING_STARTED_URL =  'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
```
Comment (kgaikwad) [inline src/constants.js:34]:
```suggestion
export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview';
```

**Pull request merged**